custom_components/opnsense/client_factory.py

```
@@ -314,6 +364,21 @@ async def _set_use_snake_case(initial: bool = False) -> None:
 
         setattr(client, "set_use_snake_case", _set_use_snake_case)
         _LOGGER.debug("Applied aiopnsense core compatibility shim for set_use_snake_case()")
+    elif not _callable_accepts_parameter(set_use_snake_case, "initial"):
+
+        async def _set_use_snake_case_with_initial(initial: bool = False) -> None:
```

Ruff (0.15.10)

[warning] Boolean-typed positional argument in function definition

(FBT001)